integrationExamples/longform/basic_wo_requireExactDuration.html

```diff
@@ -0,0 +1,226 @@
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="//netdna.bootstrapcdn.com/font-awesome/3.0.2/css/font-awesome.css" />
+  <!-- <script type="text/javascript" src="//adm.fwmrm.net/p/vitest-js/AdManager.js"></script> -->
+  <script type="text/javascript" src="https://mssl.fwmrm.net/libs/adm/6.24.0/AdManager-debug.js"></script>
+  <!-- <script type="text/javascript" src="player.js"></script> -->
+  <style type="text/css">
+    #videoPlayer {
+      height: 480px;
+      width: 100%;
+      background-color: #000;
+    }
+
+    .outer {
+      position: relative;
+    }
+
+    #skip {
+      position: absolute;
+      right: 20px;
+      bottom: 20px;
+      display: none;
+      color: white;
+      background-color: black;
+      border: 1px solid gray;
+    }
+
+    #adCount {
+      color: white;
+      position: absolute;
+      left: 20px;
+      bottom: 20px;
+      display: none;
+    }
+
+    #showad {
+      color: white;
+      position: absolute;
+      left: 20px;
+      top: 20px;
+    }
+  </style>
+  <script>
+    var pbjs = pbjs || {};
+    pbjs.que = pbjs.que || [];
+
+    var videoAdUnit = [{
+      code: 'sample-code',
+      sizes: [640, 480],
+      mediaTypes: {
+        video: {
+          context: 'adpod',
+          playerSize: [640, 480],
+          adPodDurationSec: 300,
+          durationRangeSec: [15, 30],
+          requireExactDuration: false
+        }
+      },
+      bids: [
+        {
+          bidder: 'appnexus',
+          params: {
+            placementId: 15394006
+          }
+        }
+      ]
+    }];
+
+    pbjs.que.push(function () {
+      pbjs.addAdUnits(videoAdUnit);
+      pbjs.setConfig({
+        debug: true,
+        cache: {
+          url: 'https://prebid.adnxs.com/pbc/v1/cache'
+        },
+        adpod: {
+          brandCategoryExclusion: true
+        }
+      });
+    });
+
+    function loadPrebidRequest() {
+      pbjs.que.push(function () {
+        pbjs.requestBids({
+          bidsBackHandler: function (bids) {
+            pbjs.adServers.freewheel.getTargeting({
+              callback: function (err, targeting) {
+                loadKv(targeting);
+                loadBids(targeting)
+                console.log(targeting);
+              }
+            });
+          }
+        });
+      });
+    }
+
+  </script>
+  <script type="text/javascript">
+    function getIndustry(id) {
```

Comment:
We should create common.js and move all functions in that file ? They are same for all the long form pages

Reply:
I will create this and update the pages.